### ISSUE TITLE
[BUG] Javascript error with reduce / enlist combination

### ISSUE THREAD

**Issue by yaisog** [Jermolene/TiddlyWiki5#7179]
### Describe the bug

This probably shouldn't be happening:
![image](https://user-images.githubusercontent.com/1133745/210500366-ec5daac1-a56b-467b-80db-20c1aaa49220.png)
Code producing the error is:
```
\define sub-filter() [enlist<doesnotexist>]

<$list filter="[reduce<sub-filter>]" />
```
As the variable name suggest, it is a non-existing variable. Just using `enlist `with that argument does not trigger an error.

### To Reproduce

1. Enter code into new tiddler on TW5 homepage.

### TiddlyWiki Configuration

- Version [5.2.5]

**Comment by saqimtiaz** [comment]
@yaisog can you please verify if this is still a problem with the current prerelease? https://tiddlywiki.com/prerelease/

**Comment by yaisog** [comment]
It seems like the prerelease is not producing the error.
Sorry if this was a dupe.

**Comment by saqimtiaz** [comment]
No worries @yaisog, it was resolved just days ago, see #7156 
Would you be so kind as please close this issue? Thank you.
